java/query/query-lambda/src/test/java/sleeper/query/lambda/WarmQueryExecutorLambdaIT.java

```diff
@@ -0,0 +1,282 @@
+import static sleeper.configuration.properties.instance.CdkDefinedInstanceProperty.QUERY_RESULTS_BUCKET;
+import static sleeper.configuration.properties.instance.CdkDefinedInstanceProperty.QUERY_RESULTS_QUEUE_URL;
+import static sleeper.configuration.properties.instance.CommonProperty.FILE_SYSTEM;
+import static sleeper.configuration.properties.instance.DefaultProperty.DEFAULT_INGEST_PARTITION_FILE_WRITER_TYPE;
+import static sleeper.configuration.properties.table.TablePropertiesTestHelper.createTestTableProperties;
+import static sleeper.configuration.properties.table.TableProperty.TABLE_NAME;
+import static sleeper.configuration.testutils.LocalStackAwsV1ClientHelper.buildAwsV1Client;
+import static sleeper.io.parquet.utils.HadoopConfigurationLocalStackUtils.getHadoopConfiguration;
+import static sleeper.query.runner.output.NoResultsOutput.NO_RESULTS_OUTPUT;
+
+@Testcontainers
+public class WarmQueryExecutorLambdaIT {
+    @Container
+    public static LocalStackContainer localStackContainer = new LocalStackContainer(
+            DockerImageName.parse(CommonTestConstants.LOCALSTACK_DOCKER_IMAGE))
+            .withServices(LocalStackContainer.Service.S3, LocalStackContainer.Service.SQS,
+                    LocalStackContainer.Service.DYNAMODB);
+
+    private final AmazonS3 s3Client = buildAwsV1Client(localStackContainer, LocalStackContainer.Service.S3,
+            AmazonS3ClientBuilder.standard());
+    private final AmazonDynamoDB dynamoClient = buildAwsV1Client(localStackContainer,
+            LocalStackContainer.Service.DYNAMODB, AmazonDynamoDBClientBuilder.standard());
+    private final AmazonSQS sqsClient = buildAwsV1Client(localStackContainer, LocalStackContainer.Service.SQS,
+            AmazonSQSClientBuilder.standard());
+    private final Configuration configuration = getHadoopConfiguration(localStackContainer);
+    private QuerySerDe querySerDe;
+
+    @TempDir
+    public Path tempDir;
+    private InstanceProperties instanceProperties;
+    protected static WarmQueryExecutorLambda lambda;
+
+    @BeforeEach
+    void setUp() throws IOException, ObjectFactoryException {
+        String dataDir = createTempDirectory(tempDir, null).toString();
+        createInstanceProperties(dataDir);
+        lambda = new WarmQueryExecutorLambda(s3Client, sqsClient, dynamoClient, instanceProperties.get(CONFIG_BUCKET));
+    }
+
+    @AfterEach
+    void tearDown() {
+        s3Client.shutdown();
+        dynamoClient.shutdown();
+        sqsClient.shutdown();
+    }
+
+    @Test
+    public void shouldCreateAQueryWithKeyTypeOfString() throws Exception {
+        // When
+        Schema schema = getStringKeySchema();
+        TableProperties tableProperties = createTestTableProperties(instanceProperties, schema);
+        createTable(tableProperties);
+        querySerDe = new QuerySerDe(schema);
+
+        // Given
+        lambda.handleRequest(new ScheduledEvent(), null);
+
+        // Then
```

Comment:
These test comments should be in the order `Given/When/Then` instead of `When/Given/Then`

Reply:
Fixed